theme/theme.config

```diff
@@ -17,6 +17,8 @@
    specify theme name below
 */
 
+@placeholder: 'default';
```

Comment:
what is this overriding?

Reply:
this fixes a build issue with less. tbh, i have no idea what it *does* but here's where I got it: https://github.com/Semantic-Org/Semantic-UI-LESS/issues/53